umh-core/vendor/github.com/getsentry/sentry-go/AGENTS.md

```
@@ -0,0 +1,106 @@
+# Sentry Go SDK
+
+Single-module Go SDK with integration sub-modules in `github.com/getsentry/sentry-go`.
+
+## Commit Attribution
+
+AI commits MUST include:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)